front/app/components/Form/formConfig.tsx

```diff
@@ -0,0 +1,97 @@
+// renderers
+import {
+  linearScaleControlTester,
+  LinearScaleControl,
+  inputControlTester,
+  InputControl,
+  textAreaControlTester,
+  TextAreaControl,
+  multiSelectCheckboxControlTester,
+  MultiSelectCheckboxControl,
+  singleSelectRadioControlTester,
+  SingleSelectRadioControl,
+  WYSIWYGControlTester,
+  WYSIWYGControl,
+  descriptionControlTester,
+  DescriptionControl,
+  topicsControlTester,
+  TopicsControl,
+  titleControlTester,
+  TitleControl,
+  imageControlTester,
+  ImageControl,
+  attachmentsControlTester,
+  AttachmentsControl,
+  clCategoryTester,
+  CLCategoryLayout,
+  locationControlTester,
+  LocationControl,
+  dateControlTester,
+  DateControl,
+  userPickerControlTester,
+  UserPickerControl,
+  multilocInputTester,
+  MultilocInputLayout,
+  orderedLayoutTester,
+  OrderedLayout,
+  checkboxControlTester,
+  singleSelectControlTester,
+  CheckboxControl,
+  MultiSelectControl,
+  multiSelectControlTester,
+  SingleSelectControl,
+} from 'components/Form/Components/Controls';
+
+export const selectRenderers = (formType: 'default' | 'input') => {
```

Comment:
Minor: It might be better to group the common renderers and just add them to the respective cases by spreading them there.

```
  const commonRenderers = [
    { tester: WYSIWYGControlTester, renderer: WYSIWYGControl },
    { tester: descriptionControlTester, renderer: DescriptionControl },
    { tester: topicsControlTester, renderer: TopicsControl },
    { tester: titleControlTester, renderer: TitleControl },
    { tester: imageControlTester, renderer: ImageControl },
    { tester: attachmentsControlTester, renderer: AttachmentsControl },
    { tester: clCategoryTester, renderer: CLCategoryLayout },
    { tester: orderedLayoutTester, renderer: OrderedLayout },
    { tester: locationControlTester, renderer: LocationControl },
    { tester: dateControlTester, renderer: DateControl },
    { tester: userPickerControlTester, renderer: UserPickerControl },
    { tester: multilocInputTester, renderer: MultilocInputLayout },
    { tester: orderedLayoutTester, renderer: OrderedLayout },
  ]
  ```
  And then spreading them where they are used. Something like 
  
  ```
  return [
        {
          tester: multiSelectCheckboxControlTester,
          renderer: MultiSelectCheckboxControl,
        },
...commonRenderers
]
  ```

Reply:
That's a good suggestion - I'll make the fix. Thanks!! :)